src/LoadTestServices/Stripe/StubCustomerService.php

```diff
@@ -21,6 +21,21 @@ public function create(array $_params): Customer
         return Customer::constructFrom($customer);
     }
 
+    public function retrieve(): Customer
+    {
+        return $this->create([]);
+    }
+
+    /**
+     * For now, keep static analysis happy and our assumptions explicit by crashing
+     * on attempts to use funding instructions while in load test stub mode. No load
+     * tests are doing this for now.
+     */
+    public function createFundingInstructions(): void
```

Comment:
If it always throws then it never returns. `never` is a subtype of every other type so this is allowed by return type covariance.

```suggestion
    public function createFundingInstructions(): never
```